src/main/kotlin/org/elm/workspace/ElmWorkspaceService.kt

```diff
@@ -315,9 +315,9 @@ class ElmWorkspaceService(
                         // Elm 0.18 requires a separate manifest/project for the test code, so we
                         // don't need to register the 'tests' directory in this case.
                     } else {
-                        // Testing on Elm 0.19 requires that there be a "tests" directory next
-                        // to the elm.json file. So we must register that as a source root if it exists.
-                        put(project.manifestPath.resolveSibling("tests"), project)
+                        // Register the directory containing the tests as a source root if it exists.
+                        // (Existence check done in `put` method above.)
```

Comment:
No big deal, but I would get rid of line 319 `// (Existence check done in 'put' method above.)`